packages/pieces/community/interfaces/src/lib/triggers/interface-trigger.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  Property,
+  TriggerStrategy,
+  createTrigger,
+} from '@activepieces/pieces-framework';
+
+export const interfaceTrigger = createTrigger({
+  name: 'interface_trigger',
+  displayName: 'Interface',
+  description: 'Trigger the flow through an interface.',
+  props: {
```

Comment:
I think the user have no idea how to reach the form link :D

Check this piece -> https://github.com/activepieces/activepieces/blob/3d4a76f00a75462a87c5be5e4040a84501bbfde2/packages/pieces/community/gravityforms/src/lib/triggers/new-submission.ts#L14